<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ChooseRasterRgb</class>
    <widget class="QWidget" name="ChooseRasterRgb">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>997</width>
                <height>635</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="1">
                <widget class="QComboBox" name="comboBoxBand_1"/>
            </item>
            <item row="2" column="0">
                <widget class="QLabel" name="label">
                    <property name="text">
                        <string>R</string>
                    </property>
                </widget>
            </item>
            <item row="3" column="1">
                <widget class="QComboBox" name="comboBoxBand_2"/>
            </item>
            <item row="4" column="1">
                <widget class="QComboBox" name="comboBoxBand_3"/>
            </item>
            <item row="6" column="1">
                <widget class="QwtPlot" name="qwtPlot_1"/>
            </item>
            <item row="3" column="0">
                <widget class="QLabel" name="label_2">
                    <property name="text">
                        <string>G</string>
                    </property>
                </widget>
            </item>
            <item row="6" column="0">
                <widget class="QPushButton" name="pushButton">
                    <property name="text">
                        <string>Choose</string>
                    </property>
                </widget>
            </item>
            <item row="4" column="0">
                <widget class="QLabel" name="label_3">
                    <property name="text">
                        <string>B</string>
                    </property>
                </widget>
            </item>
            <item row="5" column="0">
                <widget class="QLabel" name="label_4">
                    <property name="text">
                        <string>Contrast enhancement</string>
                    </property>
                </widget>
            </item>
            <item row="5" column="1">
                <widget class="QComboBox" name="comboBoxContrast"/>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QwtPlot</class>
            <extends>QFrame</extends>
            <header>qwt_plot.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
